SeaPublicWebsite/Views/Shared/_Layout.cshtml

```diff
@@ -52,8 +52,8 @@
         {
             ServiceName = "Simple Energy Advice service",
             AssetsPath = assetsPath,
-            HomepageUrl = "/",
-            ServiceUrl = "/",
+            HomepageUrl = "https://www.gov.uk",

```

Comment:
Does this break the link on local sites? Should it go in appsettings so we can override it?

Reply:
HomepageUrl is the link that the left part of the banner (crown GOV.UK) redirects to. For what it's worth, the gender pay gap project did it similarly.
ServiceUrl is the link that the text of the banner redirects to. We can either make it redirect to /energy-efficiency, or to '/' since the home controller will redirect it anyway. What do you think?

Reply:
Ah Ok, I was assuming that `HomepageUrl` would be our service's homepage